### ISSUE TITLE
"internal_Isend" error in HSE jobs

### ISSUE THREAD

**Issue by pxlxingliang** [deepmodeling/abacus-develop#3674]
### Describe the bug

In below example, I try to do the HSE calculation, but get the error:
```
Abort(671729666) on node 0 (rank 0 in comm 0): Fatal error in internal_Isend: Unknown error class, error stack:
internal_Isend(14967): MPI_Isend(buf=0x1509aba11010, count=-799087360, MPI_CHAR, 0, 0, MPI_COMM_WORLD, request=0x1917e43a0) failed
```

[001.zip](https://github.com/deepmodeling/abacus-develop/files/14475924/001.zip)

### Task list for Issue attackers (only for developers)

- [ ] Verify the issue is not a duplicate.
- [ ] Describe the bug.
- [ ] Steps to reproduce.
- [ ] Expected behavior.
- [ ] Error message.
- [ ] Environment details.
- [ ] Additional context.
- [ ] Assign a priority level (low, medium, high, urgent).
- [ ] Assign the issue to a team member.
- [ ] Label the issue with relevant tags.
- [ ] Identify possible related issues.
- [ ] Create a unit test or automated test to reproduce the bug (if applicable).
- [ ] Fix the bug.
- [ ] Test the fix.
- [ ] Update documentation (if necessary).
- [ ] Close the issue and inform the reporter (if applicable).

**Comment by WHUweiqingzhou** [comment]
@PeizeLin ,
could you have a look?

**Comment by PeizeLin** [comment]
The bug is same as issue #2934 .
The latest LibComm fixes this bug.

**Comment by pxlxingliang** [comment]
@PeizeLin Would you please update the linked LibComm commit in abacus?

**Comment by PeizeLin** [comment]
> @PeizeLin Would you please update the linked LibComm commit in abacus?

There's still a strange bug remained in the latest LibComm. I want to find out the bug before update in abacus.

**Comment by WHUweiqingzhou** [comment]
This bug has been solved by the latest LibComm, while the latest version has other unsolved issue. So we cannot use the it now.

**Comment by PeizeLin** [comment]
The bug has been fixed in the 2023.11.06 version of LibComm.
